src/osemgrep/configuring/Semgrep_envvars.ml

```diff
@@ -54,15 +54,19 @@ let in_env var = env_opt var <> None
 type t = {
   semgrep_url : Uri.t;
   fail_open_url : Uri.t;
+  metrics_url : Uri.t;
   app_token : string option;
+  integration_name : string option;
```

Comment:
maybe a comment explaining what is this integration_name?

Reply:
I'm actually not very familiar with this one (think it's some labeling that may not be widely used). tried to collect stray env vars and put them in there new home here.